tecken/upload/models.py

```diff
@@ -52,13 +52,21 @@ class FileUpload(models.Model):
 
     NOTE! Generally these objects are created in bulk.
     """
-    upload = models.ForeignKey(Upload)
+    upload = models.ForeignKey(Upload, null=True)
     bucket_name = models.CharField(max_length=100)
     key = models.CharField(max_length=300)
     # True if this overwrote an existing key
     update = models.BooleanField(default=False)
     # True if the file was gzip compressed before being uploaded
     compressed = models.BooleanField(default=False)
     size = models.PositiveIntegerField()
+    # Used when this upload belongs to a Microsoft proxy download
+    microsoft_download = models.BooleanField(default=False)
```

Comment:
This way we can later, easily, query how many/which symbol uploads came from this Microsoft background task thing.